app.vue

```diff
@@ -1,95 +1,8 @@
-                        Multiple Languages
-                    </span>
-                    <span class="portrait:text-xs text-gray-700 text-center">
-                        We're here to connect you to the services you need, in the language you're comfortable with
-                    </span>
-                </div>
-                <div class="flex flex-col w-96 items-center">
-                    <svg role="img" alt="heart" title="heart" class="h-20 w-20 self-center text-primary pt-2">
-                        <use xlink:href="./assets/images/heart-icon.svg#heart-icon-svg" />
-                    </svg>
-                    <span class="landscape:text-2xl portrait:text-xl font-extrabold py-6 font-['notosans-bold']">
-                        Free & Reliable
-                    </span>
-                    <span class="portrait:text-xs text-gray-700 text-center">
-                        We're dedicated to providing free and reliable healthcare resources so you can stay informed.
-                    </span>
-                </div>
-            </div>
-
-            <div class="my-8 flex items-center justify-center bg-[url('/assets/images/map-background.png')] h-80">
-                <button class="rounded-full bg-currentColor/90 text-white px-6 py-3">
-                    <a href="https://docs.google.com/forms/d/e/1FAIpQLSesYIL33Q_qB4ftwmEqRfZBR_tzC8vAgju4fDaV0peEdVZ2bQ/viewform"
-                        target="_blank">
-                        Get notified when we are ready!
-                    </a>
-                </button>
-            </div>
-
-            <div class="my-8 flex flex-1 justify-center">
-                <!-- <svg role="img" alt="heart circle icon" title="heart circle icon"
-                    class="self-center justify-self-center h-[412px] w-[392px] fill-none">
-                    <use xlink:href="./assets/images/heart-circle.svg#heart-circle-svg" />
-                </svg> -->
-                <img src="@/assets/images/heart-circle.png" class="h-[392px] w-[412px]" />
-            </div>
-
-            <div class="footer flex grow-1 shrink-0 self-stretch items-stretch">
-                <div class="middle-footer section flex flex-1 align-bottom justify-center text-gray-500">
-                    <span class="self-center">Powered by</span>
-                    <nav class="flex mx-2 self-center">
-                        <NuxtLink to="https://www.netlify.com/" target="_blank"
-                            class="underline hover:text-primary-hover transition-colors" data-testid="netlify-link">
-                            Netlify
-                        </NuxtLink>
-                    </nav>
-                </div>
-            </div>
-        </div>
+    <div class="h-screen flex flex-col font-sans text-primary-text bg-primary-bg theme-violet">
+        <TopNav />
+        <NuxtPage class="flex flex-1" />
```

Comment:
I need to double check, but if I remember correctly, I was trying to remove the `NuxtPage` from `app.vue` for some reason 🤔 everything else looks good

Reply:
@ermish This is just the original file renamed from app copy.vue. Let's figure out the `NuxtPage` later?